### ISSUE TITLE
React 17 peer dependency

### ISSUE THREAD

**Issue by justinlittman** [hc-oss/react-multi-select-component#479]
React 17 is listed as a peer dependency of react-multi-select-component. With NPM 7, peer dependencies are now installed by default. In the case of my React 16 app, this results in:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: sinopia_editor@3.7.1
npm ERR! Found: react@16.14.0
npm ERR! node_modules/react
npm ERR!   react@"^16.14.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@">=17" from react-multi-select-component@4.1.11
npm ERR! node_modules/react-multi-select-component
npm ERR!   react-multi-select-component@"^4.1.11" from the root project
npm ERR!
```

Does the peer dependency need to be React 17 or can the React version be lowered?

(Thanks for considering.)

**Comment by harshzalavadiya** [comment]
I guess 16+ should be fine, I'll downgrade peer dependency to 16+

**Comment by justinlittman** [comment]
Thanks!

**Comment by harshzalavadiya** [comment]
@justinlittman I have published `v4.1.12` that should allow react 16+ as well

also updating all dependencies to 17+ is highly recommend :smile:
